guideforms.py

```diff
@@ -256,25 +262,147 @@
+         'certain thing cannot be tested (<a '
+         'href="https://github.com/w3c/web-platform-tests/issues/3867">'
+         'example</a>), a spec issue for some change that would make it '
+         'possible to test. (<a href="'
+         'https://github.com/whatwg/fullscreen/issues/70">example</a>), or '
+         'a Chromium issue to upstream some existing tests (<a href="'
+         'https://bugs.chromium.org/p/chromium/issues/detail?id=695486">'
+         'example</a>).')),
+
+    'sample_links': forms.CharField(
+        label='Samples links', required=False,
+        widget=forms.Textarea(attrs={'cols': 50, 'maxlength': 500}),
+        help_text='Links to samples (one URL per line).'),
+
+    'bug_url': forms.URLField(
+        required=False, label='Tracking bug URL',
+        help_text=
+        ('Tracking bug url (https://bugs.chromium.org/...). This bug '
+         'should have "Type=Feature" set and be world readable.')),
+
+    'blink_components': forms.ChoiceField(
+      required=True,
+      label='Blink component',
+      help_text=
+      ('Select the most specific component. If unsure, leave as "%s".' %
+       models.BlinkComponent.DEFAULT_COMPONENT),
+      choices=[(x, x) for x in models.BlinkComponent.fetch_all_components()],
+      initial=[models.BlinkComponent.DEFAULT_COMPONENT]),
+
+    'devrel': forms.CharField(
+        required=False, label='Developer relations emails',
+        help_text='Comma separated list of full email addresses.'),
+
+    'impl_status_chrome': forms.ChoiceField(
+        required=True, label='Status in Chromium',
+        choices=models.IMPLEMENTATION_STATUS.items()),
+
+    'shipped_milestone': forms.IntegerField(
+        required=False, label='',
+        help_text='Desktop:<br/>' + SHIPPED_HELP_TXT),
+
+    'shipped_android_milestone': forms.IntegerField(
+        required=False, label='',
+        help_text='Chrome for Android:</br/>' + SHIPPED_HELP_TXT),
+
+    'shipped_ios_milestone': forms.IntegerField(
+        required=False, label='',
+        help_text='Chrome for iOS (RARE):<br/>' + SHIPPED_HELP_TXT),
+
+    'shipped_webview_milestone': forms.IntegerField(
+        required=False, label='',
+        help_text='Android WebView:<br/>' + SHIPPED_HELP_TXT),
+
+    'prefixed': forms.BooleanField(
+        required=False, initial=False, label='Prefixed?'),
+
+    'footprint': forms.ChoiceField(
+        label='Technical footprint',
+        choices=models.FOOTPRINT_CHOICES.items(),
+        initial=models.MAJOR_MINOR_NEW_API),
+
+    'visibility': forms.ChoiceField(
+      label='Developer visibility',
+      choices=models.VISIBILITY_CHOICES.items(),
+      initial=models.WARRANTS_ARTICLE,
+      help_text=('How much press / media / web developer buzz will this '
+                 'feature generate?')),
+
+    'search_tags': forms.CharField(
+        label='Search tags', required=False,
+        help_text='Comma separated keywords used only in search'),
```

Comment:
For now, it doesn't seem to work at all.  I can look into fixing it for the search box on this site.  Also, eventually we will add these search tags to the feature detail page so that search engines will see them too.